pubspec.yaml

```diff
@@ -163,7 +179,11 @@ dependencies:
   async: ^2.11.0
   cached_network_image: ^3.2.3
   flutter_image_compress: ^2.0.4
-  image_gallery_saver: ^2.0.3
+  # TODO: remove when https://github.com/hui-z/image_gallery_saver/pull/310 is merged
+  image_gallery_saver:
```

Comment:
- [ ] demo functions related to image_saver

Reply:
iOS:

https://github.com/user-attachments/assets/55fe77ab-f109-44c1-8b68-9bd0b692316a

Reply:
https://github.com/user-attachments/assets/d24128bf-37c9-4b58-9fd8-f2be2efde1bb